common/web/types/package.json

```diff
@@ -12,6 +12,10 @@
     ".": {
       "es6-bundling": "./src/main.ts",
       "default": "./build/src/main.js"
+    },
+    "./utils": {
+      "es6-bundling": "./src/util/index.ts",
+      "default": "./build/src/util/index.js"
```

Comment:
What is the purpose of this? I don't want to see separate exports for the module -- it'd be better to split it into a separate module than to do this.

Reply:
No longer needed. Removed.